Dockerfile

```diff
@@ -18,28 +20,20 @@ COPY extra_playbooks ./extra_playbooks
 COPY playbooks ./playbooks
 COPY plugins ./plugins
 
-RUN apt update -q \
-    && apt install -yq --no-install-recommends \
```

Comment:
Why apt-get -> apt ? Anyway that's unrelated to the pip stuff, should probably cut this out.